FlowCrypt/src/androidTest/java/com/flowcrypt/email/matchers/BackgroundTintMatcher.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ * © 2016-present FlowCrypt a.s. Limitations apply. Contact human@flowcrypt.com
+ * Contributors: DenBond7
+ */
+
+package com.flowcrypt.email.matchers
+
+import android.content.Context
+import android.content.res.ColorStateList
+import android.view.View
+import android.widget.TextView
+import androidx.test.espresso.matcher.BoundedDiagnosingMatcher
+import org.hamcrest.Description
+
+/**
+ * @author Denis Bondarenko
+ *         Date: 4/21/22
+ *         Time: 1:31 PM
+ *         E-mail: DenBond7@gmail.com
+ */
+class BackgroundTintMatcher(private val expectedBackgroundTintList: ColorStateList) :
+  BoundedDiagnosingMatcher<View?, TextView>(TextView::class.java) {
+  private var context: Context? = null
+
+  override fun matchesSafely(textView: TextView, mismatchDescription: Description): Boolean {
+    context = textView.context
+    val actualBackgroundTintList = textView.backgroundTintList
+    mismatchDescription
```

Comment:
Shouldn't the `mismatchDescription` be updated only if mismatch has actually happened?

Reply:
`BackgroundTintMatcher` based on [ViewMatchers.hasTextColor()](https://android.googlesource.com/platform/frameworks/testing/+/android-support-test/espresso/core/src/main/java/android/support/test/espresso/matcher/ViewMatchers.java#1179). I've reinvestigated sources and now I'm not sure that `mismatchDescription` is used at all. I will remove that code.